Easydict/NewApp/Model/TTSService.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  TTSService.swift
+//  Easydict
+//
+//  Created by 戴藏龙 on 2024/1/13.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import Defaults
+import Foundation
+
+@available(macOS 13, *)
+enum TTSService: String, CaseIterable, CustomLocalizedStringResourceConvertible {
```

Comment:
Can we rename `TTSService` to `TTSServiceType`?  Since it's a service type here, not a service.